contracts/UpgradeableAssetToken.sol

```diff
@@ -0,0 +1,331 @@
+// Copyright (c) 2017-2018 Clearmatics Technologies Ltd
+
+// SPDX-License-Identifier: LGPL-3.0+
+
+pragma solidity ^0.5.0;
+
+/**
+    same libraries now imported from openzeppelin-eth
+    all of them are upgradeable by default
+*/
+import "zos-lib/contracts/Initializable.sol";
+import "openzeppelin-eth/contracts/token/ERC20/IERC20.sol";
+import "openzeppelin-eth/contracts/math/SafeMath.sol";
+import "./ERC223Interface.sol";
+import "./ERC223ReceivingContract.sol";
+
+
+contract UpgradeableAssetToken is Initializable, ERC223Interface, IERC20 {
```

Comment:
I would replace the existing none upgradable asset token with this code. We don't expect to support the none upgradable version now we have this version